frontend/src/pages/chat/components/chat-footer/chat-footer.tsx

```diff
@@ -0,0 +1,56 @@
+import { Button, Input } from '#libs/components/components.js';
+import { Icon } from '#libs/components/icon/icon.js';
+import { DEFAULT_INPUT } from '#libs/constants/constants.js';
+import { typer } from '#libs/helpers/helpers.js';
+import { useAppForm, useCallback } from '#libs/hooks/hooks.js';
+import { type FormControl, type FormFieldValues } from '#libs/types/types.js';
+
+import styles from './styles.module.scss';
+
+type Properties = {
+  onSend: ({ text }: { text: string }) => void;
+};
+
+type HandleForm = () => void;
+
+type FormValue = {
+  text: string;
+} & FormFieldValues;
```

Comment:
```suggestion
type FormValue = {
  text: string;
};
```
And move this type to a separate file